### DISCUSSION TITLE
VDI 안과 메타몽에 스크롤이 같이 동작하는 현상

### DISCUSSION THREAD

**Discussion by DolDolDol** [metamong-friends/metamong-friends.github.io#23] [제보하기]
조건 : 디스플레이 1에 메타몽이, 디스플레이 2에 VDI가 전체 모드로 떠 있다. 

1. 메타몽을 클릭해서 포커스가 가도록 한다.
2. VDI 위로 마우스 커서를 올리고 마우스 휠로 스크롤 한다.
=> VDI안과 메타몽에 같이 마우스 휠 스크롤 이벤트가 발생된다.

**Comment by ACT-TJ** [comment]
요거 지금 해보는데 모니터 1-2 / 2-1 바꿔보거나 해도 재현이 잘 되지않네요 ㅠ

재택도 멀티 디스플레이를 쓰시는 지.. 혹 시간되실 때 한번만 더 확인을 부탁드리겠고,
오프라인 근무하실 때 자리 찾아뵙고 또 재현 특이사항 여쭤보도록 할게요!!
